src/blueprints/Do.Blueprints.Service.Application/Authentication/AuthenticationExtensions.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Do;
 
-public static class AuthenticationExtensions
+public static partial class AuthenticationExtensions
```

Comment:
instead of making this partial, let's move all extensions to their related feature/layer extension class.

move extensions like `int`, `guid` etc to `CoreExtensions`